test_scripts/Resumption/Handling_errors_from_HMI/021_resume_all_data_with_invalid_response_to_one_rpc_unexpected_disconnect.lua

```diff
@@ -0,0 +1,62 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0190-resumption-data-error-handling.md
+--
+-- Description:
+-- In case:
+-- 1. AddCommand, AddSubMenu, CreateInteractionChoiceSet, SetGlobalProperties, SubscribeButton, SubscribeVehicleData,
+--  SubscribeWayPoints, CreateWindow (<Rpc_n>) are added by app
+-- 2. Unexpected disconnect and reconnect are performed
+-- 3. App re-registers with actual HashId
+-- SDL does:
+--  - start resumption process
+--  - send set of <Rpc_n> requests to HMI
+-- 4. HMI sends invalid response to one request and <successful> for others
+-- SDL does:
+--  - ignore invalid response from HMI and process others
+--  - remove already restored data
+--  - send set of revert <Rpc_n> requests to HMI (except the one related to invalid response)
+--  - respond RegisterAppInterfaceResponse(success=true,result_code=RESUME_FAILED) to mobile application
+--    when default timeout expires
```

Comment:
remove this line

Reply:
Since the response is invalid SDL ignores it. However SDL is still waiting a correct response will be sent by HMI within default timeout. So this line shouldn't be removed.